websites/G/GoAnimes/presence.ts

```diff
@@ -0,0 +1,87 @@
+const presence = new Presence({
+		clientId: "1161309082179883089",
+	}),
+	onBrowsingTimeStamp = Math.floor(Date.now() / 1000);
+
+let currentTime: number,
+	duration: number,
+	paused: boolean,
+	played: boolean,
+	timestamps;
+
+const searchTxt: HTMLInputElement = document.querySelector("#s"),
+	paginationTxt: HTMLElement = document.querySelector(
+		"#contenedor > div.module > div.content > div.pagination > span:nth-child(1)"
+	),
+	animenameTxt: HTMLElement = document.querySelector(
+		"#single > div.content > div.sheader > div.data > h1"
+	),
+	airdate: HTMLElement = document.querySelector(
+		"#single > div.content > div.sheader > div.data > div.extra > span.date"
+	),
+	episodenameTxt: HTMLElement = document.querySelector("#info > h1");
+
+interface IFrameMetaData {
+	currentTime: number;
+	duration: number;
+	paused: boolean;
+	played: boolean;
+}
+
+presence.on("iFrameData", (data: IFrameMetaData) => {
+	({ currentTime, duration, paused, played } = data);
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/WJzODPK.png",
+		startTimestamp: onBrowsingTimeStamp,
+	};
+
+	if (document.location.pathname === "/") {
```

Comment:
Considering that it is used many times in this presence, it may be more readable to extract the property.
```suggestion
	const { pathname } = document.location;
	if (pathname === "/") {
```